components/tabs/tabs.js

```diff
@@ -332,10 +332,9 @@ class Tabs extends LocalizeStaticMixin(ArrowKeysMixin(RtlMixin(LitElement))) {
 
 		return html`
 			<div class="${classMap(tabsLayoutClasses)}">
-				<div ?allow-scroll-next="${this._allowScrollNext}"
-					?allow-scroll-previous="${this._allowScrollPrevious}"
+				<div ?data-allow-scroll-next="${this._allowScrollNext}"
+					?data-allow-scroll-previous="${this._allowScrollPrevious}"
 					class="d2l-tabs-container"
-					?scroll-collapsed="${this._scrollCollapsed}"
```

Comment:
This doesn't appear to have been referenced by anything.

Reply:
Good find - I suspect impl for this behavior pivoted and this never got cleaned up.